scripts/check_latest-dev_updates.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import sys
+import re
+
+
+# also need a function to separate comments from actual commands
+def remove_comments(lines):
+    code_lines = []
+    for line in lines:
+        if not line.startswith("--"):
+            code_lines.append(line)
+    return "\n".join(code_lines)
+
+
+def unsafe_catalog_modification(s):
+    # catalog tables reside in _timescaledb_catalog, _timescaledb_internal and _timescaledb_config
+    # it is unsafe to simply ADD or DROP columns from catalog tables
+    matches = re.search(
+        r"ALTER\s*TABLE\s*(_timescaledb_catalog|_timescaledb_internal|_timescaledb_config)[\s\S]+(DROP|ADD)\s*COLUMN",
```

Comment:
I would like to understand the reason for using `\s*` and not `\s+` here. Is there anything specific we want to capture with the current regex that the following one does not capture?

```
 r"ALTER\s+TABLE\s+(_timescaledb_catalog|_timescaledb_internal|_timescaledb_config)[\s\S]+(DROP|ADD)\s+COLUMN",
```

Reply:
Yes this should be s+ as you have pointed out. No reason to use s* here because at least a space is required there between DROP/ADD and COLUMN